_data/navigation.yml

```diff
@@ -1,14 +1,27 @@
 # main links links
 main:
-  - title: "About"
-    url: https://mmistakes.github.io/minimal-mistakes/about/
-  - title: "Blog"
-    url: /year-archive/
+  #- title: "About"
+  #  url: https://mmistakes.github.io/minimal-mistakes/about/
+  #- title: "Blog"
+  #  url: /year-archive/
+  #- title: "Downloads"
+  #  url: /collection-archive/
+  # - title: "Contribute"
+  #   url: /contribute-website/
+  - title: "About Us"
+    url: /about/
+  - title: "News and Events"
+    url: /news/
+  - title: "Projects"
+    url: /projects/
   - title: "Downloads"
-    url: /collection-archive/
-  - title: "Contribute"
-    url: /sitemap/
-
+    url: /downloads/
+  - title: "Driverless Printing"
+    url: /driverless/
+  - title: "Contact Us"
+    url: /contact/
+  - title: "Donation"
```

Comment:
Please change "Donation" to "Donations"
also, it would make sense to change the corresponding link to "donations"